components/eam/bld/configure

```diff
@@ -2660,10 +2676,10 @@ sub write_filepath
             die "configure ERROR: invalid option - crm_adv: $crm_adv.\n";
           }
                               
-        } elsif ($crm eq 'samxx') {
+        } elsif (($crm eq 'samxx') || ($crm eq 'pam')) {
```

Comment:
is this intended to be an empty in elsif body?

Reply:
yes, just to hold the comment about not needing this section. I guess it's kinda silly to add the else if just for this though. Not sure why it was done this way originally. I'll change it.

Reply:
done